lib/dates_from_string/patterns.rb

```diff
@@ -32,6 +32,7 @@ module Patterns
 
   DATE_COUNTRY_FORMAT = {
     default: -> { [:year, :month, :day] },
+    'pt-BR': -> { [:day, :month, :year] },
```

Comment:
Why you not use Current ISO 3166 country codes standard, where Brazil is BRA.
```suggestion
    bra: -> { [:day, :month, :year] },
```

Reply:
For me that's ok.